midterm-2023-11-07/code/5042-ivan-apostolovski/script.ts

```diff
@@ -0,0 +1,358 @@
+interface Movie {
+    id: number
+    title: string
+    year: number
+    director: string
+    genre: string[]
+    plot: string
+    cast: Cast[]
+    oscars: any
```

Comment:
The data is in a format that is common in JavaScript, so it needs to be typed better than any (actually it should be a `Record<string, string>`)